client/src/templates/layouts/NavigationLayout.js

```diff
@@ -263,10 +264,18 @@ const StyledDrawer = styled(Drawer)`
 
 const NavigationLayout = (props) => {
   const { t } = useTranslation();
-  const { authLoading, mobiletabs, navSearch, tabIndex, isAuthenticated, user } = props;
+  const {
+    authLoading,
+    mobiletabs,
+    navSearch,
+    tabIndex,
+    isAuthenticated,
+  } = props;
   const history = useHistory();
   const [drawerOpened, setDrawerOpened] = useState(false);
   const [searchKeywords, setSearchKeywords] = useState(false);
+  const { userProfileState, userProfileDispatch } = useContext(UserContext);
```

Comment:
```suggestion
```

But for the solution we should stick with `user` which comes from https://github.com/FightPandemics/FightPandemics/blob/1a471543dca12ccb00a7b63367ab3ef58356e2de/client/src/reducers/sessionReducer.js#L47. 

`UserContext` here is very confusing name wise, it should be called ProfileContext and shouldn't be used outside of org/user profile pages. Not sure when/why we diverted on state management but we definitely should be aiming for something more clear & consistent. A goal for post wide launch...